star-ros2/src/star_simulation/test/test_ekf_drift.py

```
@@ -0,0 +1,120 @@
+
+        cls.raw_sub = cls.node.create_subscription(
+            Odometry, '/odom/unfiltered', cls._raw_cb, 10)
+        cls.filtered_sub = cls.node.create_subscription(
+            Odometry, '/odometry/filtered', cls._filtered_cb, 10)
+
+    @classmethod
+    def _raw_cb(cls, msg):
+        cls.last_raw = msg
+
+    @classmethod
+    def _filtered_cb(cls, msg):
```

Ruff (0.15.9)

[warning] Missing return type annotation for classmethod `_filtered_cb`

Add return type annotation: `None`

(ANN206)